src/pages/library/common.js

```diff
@@ -519,7 +518,7 @@ export const SearchAndFilterComponent = ({
       ]),
       div({ style: { marginLeft: '1rem', minWidth: 0, width: '100%', height: '100%' } }, [
         _.isEmpty(filteredData.data) ? div({ style: { margin: 'auto', textAlign: 'center' } }, ['No Results Found']) :
-          children({ filteredList: filteredData.data, sections, selectedTags, setSelectedTags })
+          children({ filteredList: filteredData.data, sections })
```

Comment:
Why don't you need to pass the selected tags down?

Reply:
Honestly I don't know.. It looks like `children` is an argument to `SearchAndFilterComponent` (like `idField` or `descField`) but I don't see anyone passing it in, so I don't see how it's actually used.

Reply:
If this was the cause of a bug, what should I do to test for that bug? The UI seems to work fine with this change when I use it.

Reply:
I think if it looks like the UI looks fine, that is fine for now. Eventually we would have unit tests which would tell us, but I think unfortunately we don't have those yet, so we have to make do somewhat with manual testing.